### PR TITLE
Playbook iochunt upgrade

### PR DESCRIPTION
**Pull request by liorkol**
And adding two subplaybooks (Crowdstsrike and Bit9)
Tested all 3

### CODE REVIEW

**Comment by dorsha** [comment]
@aviadl - please review.

File: Scripts/script-ReadFile.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: eccea40d-921c-4a52-875a-738855531f58
```
Comment (aviadl) [inline Scripts/script-ReadFile.yml]: static ID
Also please limit by size, if someone will load a few Giga file here we would die

File: Scripts/script-ReadFile.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+name: ReadFile
+script: |-
+  res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
+  with open(res[0]['Contents']['path'], 'r') as f:
+      data = f.read()
+  if data:
+      message = 'Read %d bytes from file.' % (len(data))
+      demisto.results({"Type" : entryTypes["note"],
+                       "ContentsFormat" : formats["text"],
+                       "Contents" : message,
```
Comment (aviadl) [inline Scripts/script-ReadFile.yml]: Put the data on the Contents, so one can extend the context with value fetched from file to other places

File: Playbooks/playbook-CrowdStrike_Rapid_IOC_Hunting.yml [hunk 3]
```diff
@@ -0,0 +1,398 @@
+id: playbook16
```
Comment (aviadl) [inline Playbooks/playbook-CrowdStrike_Rapid_IOC_Hunting.yml]: version should be -1 (on all tasks)
Change all conditions to else

File: Playbooks/playbook-Carbon_black_Protection_Rapid_IOC_Hunting.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
+id: playbook17
```
Comment (aviadl) [inline Playbooks/playbook-Carbon_black_Protection_Rapid_IOC_Hunting.yml]: version should be -1

**Pull request merged**